Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/Internal/FunctionConfiguration.cs

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) Endjin Limited. All rights reserved.
 // </copyright>
 
-namespace Corvus.SpecFlow.Extensions
+namespace Corvus.Testing.AzureFunctions.Internal
```

Comment:
This shouldn't be in the `Internal` namespace, as it's intended for use by external code - it should be in `Corvus.Testing.AzureFunctions`.